unattended_installer/install_functions/checks.sh

```diff
@@ -135,23 +135,14 @@ function checks_arguments() {
     # -------------- Wazuh ------------------------------------------
 
     if [ -n "${wazuh}" ]; then
-        if [ -n "${wazuh_installed}" ] || [ -n "${wazuh_remaining_files}" ]; then
+        if [ -n "${wazuh_installed}" ] || [ -n "${wazuh_remaining_files}" ] || [ -n "${filebeat_installed}" ] || [ -n "${filebeat_remaining_files}" ]; then
             if [ -n "${overwrite}" ]; then
                 installCommon_rollBack
             else
                 common_logger -e "Wazuh is already installed in this node or some of its files haven't been erased. Use option -o|--overwrite to overwrite all components."
```

Comment:
What about if a user receives this message and wants to manually remove the component? it's not possible with the information presented.

Reply:
Done [9822e92](https://github.com/wazuh/wazuh-packages/pull/1663/commits/9822e923dc31c00b848cf303afa79abc58707ea5)